### PR TITLE
Attach results to the reporter

### PR DESCRIPTION
**Pull request by geek**
Relates to #389

### CODE REVIEW

**Comment by geek** [comment]
@hofan41 what do you think of this change?

**Comment by hofan41** [comment]
Looks good to me! Are there particular rules between when to use internals and when to use self/this?

**Comment by geek** [comment]
It was more a concern of leaving the results on the static internals for the next time multiple is created... vs keeping the results with the instance.

**Comment by hofan41** [comment]
Yeah I notice that now too. Good catch!

File: lib/reporters/multiple.js [hunk 1]
```diff
@@ -4,10 +4,12 @@ var Path = require('path');
...
 
+
 // Declare internals
 
 var internals = {};
 
+
 exports = module.exports = internals.Reporter = function (options) {
 
     internals.reporters = [];
```
Comment (hofan41) [inline lib/reporters/multiple.js]: Shouldn't we do the same with internals.reporters?

**Pull request merged**